tclap/include/tclap/StdOutput.h

```diff
@@ -164,30 +164,35 @@ inline void StdOutput::_shortUsage( CmdLineInterface& _cmd,
   XorHandler xorHandler = _cmd.getXorHandler();
   std::vector< std::vector<Arg*> > xorList = xorHandler.getXorList();
 
-  std::string s = progName + " ";
+  std::string s = progName + "";
```

Comment:
Was removal of the space between the quotation marks intentional? If yes, shouldn't we remove the whole literal (just keep `std::string s=progName;`)?

Reply:
@lassoan I was trying to minimize the differences from the original tclap code and sttill improve the resulting formatting style.  I will remove in the next revision.